src/screens/Account/LendingBanners.js

```diff
@@ -28,6 +33,9 @@ export default function LendingBanners({ account }: Props) {
   const compoundCapabilities = availableOnCompound
     ? getAccountCapabilities(account)
     : {};
+  const compoundSummary =
+    availableOnCompound &&
+    makeCompoundSummaryForAccount(account, parentAccount);
```

Comment:
i'm not sure if it passes Flowtype, but it is better for flow to do `availableOnCompound  ? make..() : null` otherwise it will tend to infer things like `CompoundSummary | boolean`

Reply:
looks like it's passing. However I get the point. I think it passes because compoundSummary is not typed ? I'll update